### ISSUE TITLE
odig support

### ISSUE THREAD

**Issue by gildor478** [ocaml/oasis#559]
__This feature request has been migrated from artifact #1701 on forge.ocamlcore.org. It was assigned to [user102](https://forge.ocamlcore.org/users/user102).__

## [user102](https://forge.ocamlcore.org/users/user102) posted on 2016-10-30 23:44:38:

It would be nice to get odig support. There is actually very little to do: https://github.com/ocaml/opam-repository/pull/7714/files#diff-1fe393d637fc0aa4170b457f702d879a

Basically:
- copy the opam file if available in lib/PKG/
- copy CHANGES, LICENSE and README if available in doc/PKG/

Documentation is obtained by running "odig help packaging".
